released/packages/coq-math-classes/coq-math-classes.1.0.7/url

```diff
@@ -0,0 +1,2 @@
+http: "https://github.com/math-classes/math-classes/archive/master.zip"
+checksum: "b8add93ba0240f6391f4060c05583115"
```

Comment:
The URL is not right because the checksum will change after each new commit. Instead you should do a release at math-classes/math-classes (you can do this online at https://github.com/math-classes/math-classes/releases) and use the corresponding tag in the URL of the OPAM package.